mainboard_flashing/common_hardware/MKS Monster8 v2/README.md

```diff
@@ -1,13 +1,20 @@
 ---
 layout: default 
+<<<<<<< HEAD:mainboard_flashing/common_hardware/MKS Monster8/README.md
```

Comment:
There is many not resolved conflicts. You should resolve them before submitting PR